rerun_py/README.md

```diff
@@ -27,14 +25,14 @@ rr.init("rerun_example_app", spawn=True)
 positions = np.vstack([xyz.ravel() for xyz in np.mgrid[3 * [slice(-5, 5, 10j)]]]).T
 colors = np.vstack([rgb.ravel() for rgb in np.mgrid[3 * [slice(0, 255, 10j)]]]).astype(np.uint8).T
 
-rr.log_points("my_points", positions=positions, colors=colors)
+rr.log("points3d", rr.Points3D(positions, colors=colors))
 ```
 
 ## Resources
-* [Quick start](https://www.rerun.io/docs/getting-started/python)
```

Comment:
We should probably update `rerun_py/docs/gen_common_index.py` also?

Reply:
what do you mean, didn't @jleibs update this file yesterday?